Week2/1_hello_world.js

```diff
@@ -0,0 +1,8 @@
+console.log("Hallo Wereld") // Dutch
```

Comment:
Goodgood, don't forget to add the `;` at the end (also for the rest of your code).

This might seem useless, but it will save you a lot of headache later on:)